project-base/storefront/components/Pages/BlogArticle/BlogArticleDetailContent.tsx

```diff
@@ -17,7 +17,7 @@ export const BlogArticleDetailContent: FC<BlogArticleDetailContentProps> = ({ bl
 
     return (
         <Webline>
-            <ArticleTitle dataTestId={TEST_IDENTIFIER + 'title'}>{blogArticle.name}</ArticleTitle>
+            <ArticleTitle dataTestId={TEST_IDENTIFIER + 'title'}>{blogArticle.seoH1 ?? blogArticle.name}</ArticleTitle>
```

Comment:
Better put this. It's because when `seoH1` would be for some reason empty string, so we are safe that this page will have some title.

```suggestion
            <ArticleTitle dataTestId={TEST_IDENTIFIER + 'title'}>{blogArticle.seoH1 || blogArticle.name}</ArticleTitle>
```

Reply:
But actually I am wondering if we really want to display seo title instead of name...? Are you sure of this change? 🤔